addons/l10n_vn/data/account.account.template.csv

```diff
@@ -159,12 +159,12 @@
 "chart6234","Depreciation expense",6234,"expense","vn_template","False"
 "chart6237","Outside services ",6237,"expense","vn_template","False"
 "chart6238","Other expenses",6238,"expense","vn_template","False"
-"chart6271","Factory staff costs",6271,"expense","vn_template","False"
-"chart6272","Material costs",6272,"expense","vn_template","False"
-"chart6273","Production tools and instruments",6273,"expense","vn_template","False"
-"chart6274","Fixed asset depreciation",6274,"expense","vn_template","False"
-"chart6277","Outside services",6277,"expense","vn_template","False"
-"chart6278","Other expenses",6278,"expense","vn_template","False"
+"chart6271","Factory staff costs",6271,"expense_direct_cost","vn_template","False"
+"chart6272","Material costs",6272,"expense_direct_cost","vn_template","False"
+"chart6273","Production tools and instruments",6273,"expense_direct_cost","vn_template","False"
+"chart6274","Fixed asset depreciation",6274,"expense_direct_cost","vn_template","False"
+"chart6277","Outside services",6277,"expense_direct_cost","vn_template","False"
+"chart6278","Other expenses",6278,"expense_direct_cost","vn_template","False"
```

Comment:
You should have a reference document about this to persuade them

Reply:
thank my friend